src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const daysOfRent = days * 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  if (days >= LONG_TERM) {
+    return daysOfRent - LONG_TERM_DISCOUNT;
+  } else if (days >= SHORT_TERM && days < LONG_TERM) {
+    return daysOfRent - SHORT_TERM_DISCOUNT;
+  }
```

Comment:
do not forget to fix it
```suggestion
  if (days >= LONG_TERM) {
    return daysOfRent - LONG_TERM_DISCOUNT;
  }
  
    if (days >= SHORT_TERM ) {
    return daysOfRent - SHORT_TERM_DISCOUNT;
  }
```